README.md

```diff
@@ -1,703 +1,48 @@
-<p align="center">
-  <img src="https://github.com/lucasbento/create-graphql/raw/master/content/logo.png">
-</p>
+# ![Create-GraphQL Logo](/media/logo.png)
 
-<h1 align="center">Create GraphQL</h1>
+# Create-GraphQL
```

Comment:
Yes, I think it would look better without the logo, but not for now. And the logo are centered without html now, because it's a fully image.